docs/en/custom.rst

```diff
@@ -20,7 +20,7 @@ Inject annotation values
 ------------------------
 
 The annotation parser checks if the annotation constructor has arguments,
-if so then we will pass the value array, otherwise will try to inject
+if so then we will pass the value array, otherwise we will try to inject
```

Comment:
shouldn't we use `it` rather than `we` to refer to the parser ?

Reply:
Agree. @greg0ire can you make that change please?

Reply:
Oh right I forgot that PR, let me change this!